examples/encoding_categorical_features.ipynb

```diff
@@ -136,6 +136,49 @@
     "ca_features.to_csv('ca_features.csv', index=False)\n",
     "ca_labels.to_csv('ca_labels.csv', index=False)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Automatic generated categorical feature encoding\n",
+    "The ```categories``` argument can automatically be generated using the ```get_definitions``` and ```generate_categories``` function to respectively download the definitions available [here](https://www2.census.gov/programs-surveys/acs/tech_docs/pums/data_dict/) and create the categories dictionary.\n",
+    "Note that this feature only works for ```survey_year```>='2017' and categories not needing the externally deffined ```PUMA``` codes. [puma codes](https://www2.census.gov/geo/docs/reference/puma/2010_PUMA_Names.txt)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from folktables import ACSIncome_categories\n",
+    "\n",
+    "data_source = ACSDataSource(survey_year='2018', horizon='1-Year', survey='person')\n",
```

Comment:
data_source and ca_data are already created a few cells above, I do not think this should be repeated.